dataQcBmd.py

```diff
@@ -117,7 +118,7 @@ def run_morpho_on_file(morph_file,full_devel='full'):
     """
     print ("morpho_input_csv_file_name:" + str(morph_file))
     #to_be_processed/7_PAH_zf_LPR_data_2021JAN11_tall.csv
-    command = "python3 /srpAnalytics/format_morpho_input.py " + \
```

Comment:
Can you please confirm that this change works in the Docker image? I worry about not having absolute paths.